lisa/git.py

```diff
@@ -67,6 +67,17 @@ def get_uncommited_patch(repo):
     """
     return git(repo, 'diff', 'HEAD')
 
+def get_commit_message(repo, ref='HEAD', format='%s'):
+    """
+    Get the reference commit message.
+
+    :returns: String containing the commit message.
+
+    :param format: Format string passed to ``git show --format``. Default to subject.
+    :type format: str
+    """
+    return git(repo, 'show', ref, '--format=' + format, '-s')
```

Comment:
maybe `ref` needs to be moved at the end, since the manpage indicates:
```
git show [<options>] [<object>...]
```
that may help avoiding conflicts between option names and branch names.

Reply:
Will do.